OverlayPlugin.Core/Controls/WSConfigPanel.cs

```diff
@@ -497,8 +523,16 @@ private void simpStartBtn_Click(object sender, EventArgs e)
 
                     if (p.WaitForExit(3000))
                     {
+                        // Ensure OutputDataReceived and ErrorDataReceived have finished processing so that ngrokTooOld is reliable
+                        p.WaitForExit();
                         simpLogBox.AppendText("ngrok crashed!\r\n");
                         UpdateTunnelStatus(TunnelStatus.Error);
+                        if (ngrokTooOld)
+                        {
+                            simpLogBox.AppendText("ngrok version is too old!\r\n");
+                            simpLogBox.AppendText("Deleting old version. Start again to redownload.\r\n");
+                            File.Delete(ngrokPath);
+                        }
```

Comment:
Is there a reasonable way to automatically retry, rather than relying on the user actually reading the error message and trying again themselves?

Reply:
Done